dev-packages/e2e-tests/test-applications/solid/tests/errorboundary.test.ts

```diff
@@ -3,7 +3,7 @@ import { waitForError } from '@sentry-internal/test-utils';
 
 test('captures an exception', async ({ page }) => {
   const errorEventPromise = waitForError('solid', errorEvent => {
-    return !errorEvent.type;
+    return !errorEvent.type && errorEvent.transaction === '/error-boundary-example';
```

Comment:
Ideally you find a way to create transactions with entirely different names, as they may leak between tests.

Reply:
Any ideas? I can only think of creating a different route per test.

Reply:
> different route per test
I think this is what we usually do 😬 another option would be to configure playwright to only sequentially run tests but it's also not ideal...

Reply:
The leaking I had happened between the errorboundary tests and the normal error tests, and there the `transaction` field is different. Maybe we can leave it as is for now and see if there's more flakiness to fix later?

Reply:
You can also set tags or contexts in the tests and assert on them!